content/contest.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Contest"
+date: 2019-04-30T23:41:52+02:00
+weight: 3
+---
+
+# Contest
+
+Due to COVID-19 measures, it's unclear whether we will be able to hold an on-site contest at the VU Amsterdam campus. We are working on both on-site and online contests and will inform you as soon as possible. Ideas have been pitched in April to the other institutions.
```

Comment:
@EuiYeonJang I think the reason for having the 2 pages was that these will divert in the information and because it will be 1 or the other it is good to make clear what is the difference so participants can not start a discussion because the rules of one of the variants were unclear.

So I'm fine with merging the pages, but than we would need to get 2 sections on that page for the rules.

Just a general comment, not a reason to not merge if the rest does agree with this.